python-flask/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.9-slim
+WORKDIR /app
+COPY . /app
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+EXPOSE 5000
+ENV FLASK_APP=wsgi.py
+CMD ["flask", "run", "--host=0.0.0.0"]
```

Comment:
I think we need to run using wgsi or gunicorn to run flask. I think Helge uses guinicorn.

Reply:
Addressed the comment. Using gunicorn to run the flask application